image-hd.c

```diff
@@ -483,7 +483,7 @@ static int hdimage_insert_gpt(struct image *image, struct list_head *partitions)
 	const char *outfile = imageoutfile(image);
 	struct gpt_header header;
 	struct gpt_partition_entry table[GPT_ENTRIES];
-	unsigned long long smallest_offset = ~0ULL;
+	unsigned long long smallest_offset = ~0ULL, extra_offset = 0;
```

Comment:
extra_offset seems to be slightly odd name. first_usable_offset, or ful_offset perhaps?

Reply:
I've changed it to `first_usable_offset`.